packages/php-table-backend-utils/src/Schema/Snowflake/SnowflakeSchemaReflection.php

```diff
@@ -21,6 +21,27 @@ final class SnowflakeSchemaReflection implements SchemaReflectionInterface
 
     private string $schemaName;
 
+    private const TYPE_NEED_FALLBACK = [
+        Snowflake::TYPE_BINARY,
+        Snowflake::TYPE_VARBINARY,
+        Snowflake::TYPE_VECTOR,
+    ];
+    /**
+     * @var array<string, array<string, array{name: string,
+     *      type: string,
+     *      kind: string,
+     *      null?: string,
+     *      default: ?string,
+     *      "primary key": string,
+     *      "unique key": string,
+     *      check: ?string,
+     *      expression: ?string,
+     *      comment: ?string,
+     *      "policy name": ?string,
+     *      "privacy domain": ?string}>>
+     */
+    private array $fallbackTableCache = [];
```

Comment:
K čemu ti bude ta cache? Budeš při refresh dělat 2x stejnů tabulku?

Reply:
Ano, může se to stát, protože v tom cyklu, co volám tu fallback metodu se procházejí sloupce. A v jedné tabulce může být více sloupců, co potřebují ten fallback. Takže teoreticky můžu řešit 2x i vícekrát stejnou tabulku.

Reply:
Jasně už to vidím :+1: je to aby to bylo raz pro jednu tabulku

Reply:
Přesně tak. Sice malá optimalizace, ale ten `DESC TABLE` fakt trvá, tak jsem to tam dal.